client/components/calculate-total.js

```diff
@@ -0,0 +1,11 @@
+import formatPrice from './format-price';
+
+function calculateTotal(array) {
+  let total = 0;
+  array.map(item => {
+    total += item.price;
+  });
```

Comment:
Don't use `array.map` if you are ignoring its return value. Use `array.forEach` instead